content/tutorial-escrow.md

```diff
@@ -19,6 +19,406 @@ When testing in [stand-alone mode](concept-stand-alone-mode.html), you can force
+// (Random hexadecimal, 72 chars in length)
+console.log(myFulfillment.getConditionBinary().toString('hex'));
+// (Random hexadecimal, 78 chars in length)
+```
+
+Save the condition and the fulfillment for later. Be sure to keep the fulfillment secret until you want to finish executing the held payment; anyone who knows the fulfillment can finish the escrow, releasing the held funds to their intended destination.
+
+
+### 2. Calculate release or cancel time
+
+A Conditional `Escrow` transaction must contain either a `CancelAfter` or `FinishAfter` field, or both. The `CancelAfter` field lets the XRP revert to the sender if the condition is not fulfilled before the specified time. The `FinishAfter` field specifies a time before which the escrow cannot execute, even if someone sends the correct fulfillment. Whichever field you provide, the time it specifies must be in the future.
+
+Example for setting a `CancelAfter` time of 24 hours in the future:
+
+<!-- MULTICODE_BLOCK_START -->
+
+_JavaScript_
+
+```js
+const rippleOffset = 946684800;
+const CancelAfter = Math.floor(Date.now() / 1000) + (24*60*60) - rippleOffset;
+console.log(CancelAfter);
+// Example: 556927412
+```
+
+_Python 2/3_
+
+```python
+from time import time
+ripple_offset = 946684800
+cancel_after = int(time()) + (24*60*60) - 946684800
+print(cancel_after)
+# Example: 556927412
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+**Warning:** In the XRP Ledger, you must specify time as seconds since the Ripple Epoch (2000-01-01T00:00:00Z). If you use a UNIX time in the `CancelAfter` or `FinishAfter` field without converting to the equivalent Ripple time first, that sets the unlock time to an extra **30 years** in the future!
+
+### 3. Submit EscrowCreate transaction
+
+[Sign and submit](reference-transaction-format.html#signing-and-submitting-transactions) an [EscrowCreate transaction][]. Set the `Condition` field of the transaction to the time when the held payment should be released. Set the `Destination` to the recipient, which can be the same address as the sender. Include the `CancelAfter` or `FinishAfter` time you calculated in the previous step.
+
+{% include 'snippets/secret-key-warning.md' %}
+
+Request:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Websocket*
+
+```json
+{% include 'code_samples/escrow/websocket/submit-request-escrowcreate-condition.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+Response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Websocket*
+
+```json
+{% include 'code_samples/escrow/websocket/submit-response-escrowcreate-condition.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+### 4. Wait for validation
+
+{% include 'snippets/wait-for-validation.md' %}
+
+### 5. Confirm that the escrow was created
+
+Use the [`tx` command](reference-rippled.html#tx) with the transaction's identifying hash to check its final status. In particular, look for a `CreatedNode` in the transaction metadata to indicate that it created an [Escrow ledger object](reference-ledger-format.html#escrow).
```

Comment:
If you were going to add a note/link about transaction metadata, it could go here, too.

Reply:
As mentioned above, putting this off for later (DOC-123).